StudentDriver/StudentDriver/Services/WebService.cs

```diff
@@ -11,41 +11,55 @@
 using Xamarin.Forms;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
+using StudentDriver.Helpers;
 
 namespace StudentDriver.Services
 {
     public static class WebService
     {
         private static string BaseUrl = "";
 
-        public static async Task<UserStats> GetStudentStats(int id)
+        public static async Task<string> GetAuthentiationToken(OAuthProvider.ProviderType providerType, string OAuthId)
```

Comment:
Maybe looking at it again, we should leave this OAuth stuff out of this PR. There is a another issue related to OAuth for mobile. #36